stripe/src/main/java/com/stripe/android/PaymentAuthWebViewStarter.kt

```diff
@@ -16,11 +16,16 @@ internal interface PaymentAuthWebViewStarter :
     AuthActivityStarter<PaymentAuthWebViewContract.Args> {
     class Legacy(
         private val host: AuthActivityStarter.Host,
+        private val isCustomTabsSupported: Boolean,
```

Comment:
It feels the logic to check if CustomTabs is supported should be encapsulated in `PaymentAuthWebViewStarter`, since we already have `AuthenActivity.Host` passed in and have access to context there, can we do `CustomTabsCapabilities(context)` inside `PaymentAuthWebViewStarter`

Reply:
🤔 `PaymentAuthWebViewStarter` is an interface. Where would you want to add it?

Reply:
is it possible to add into the `Legacy` class ? that way the `PaymentAuthWebViewStart` won't need to check that outside

Reply:
I'd rather not depend more on `AuthActivityStarter.Host`, it's a pretty brittle class as is 😬